### PR TITLE
Added endpoint to set the clear color

### PR DESCRIPTION
**Pull request by benjamaan476**
Related to https://github.com/KittyCAD/engine/issues/1796

### CODE REVIEW

File: modeling-cmds/src/ok_response.rs [hunk 1]
```diff
@@ -160,6 +160,12 @@ define_ok_modeling_cmd_response_enum! {
             /// Curve type
             pub curve_type: CurveType,
         }
+        /// The response from the `SetClearColor` command.
+        #[derive(Debug, Serialize, Deserialize, JsonSchema, Eq, PartialEq, ExecutionPlanValue, ModelingCmdOutput)]
+        pub struct SetClearColor {
+            /// Successfully set the clear color
+            pub success: bool,
```
Comment (adamchalmers) [inline modeling-cmds/src/ok_response.rs:167]: There's no need for a response here, all this response is doing is indicating if there was an error or not, and we already show errors to the user. If the command fails, the user will get an error the usual way through our error schema. Take a look at how the other commands respond with errors.

File: modeling-cmds/src/def_enum.rs [hunk 2]
```diff
@@ -633,6 +633,13 @@ define_modeling_cmd_enum! {
             pub adjust_camera: bool,
         }
 
+        /// Set the clear color of the scene.
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:636]: Could you please explain what "the clear color" means?

**Review by adamchalmers — CHANGES_REQUESTED** [review]

File: modeling-cmds/src/ok_response.rs (hunk 1 above)
Comment (benjamaan476) [reply modeling-cmds/src/ok_response.rs]: thanks for explaining this. I have removed it now

**Review by adamchalmers — APPROVED** [review]

File: modeling-cmds/src/def_enum.rs (hunk 2 above)
Comment (benjamaan476) [reply modeling-cmds/src/def_enum.rs]: I have changed this to SetBackgroundColor as a more descriptive name

**Pull request merged**